tests/appsec/iast/sink/test_path_traversal.py

```diff
@@ -21,16 +21,13 @@ class TestPathTraversal(BaseSinkTest):
         "python": {"flask-poc": "app.py", "django-poc": "app/urls.py"},
     }
 
-    @missing_feature(library="nodejs", reason="Endpoint not implemented")
     def test_secure(self):
```

Comment:
you can entirely remove the `def test_secure(self):` : if it does not carry any decorator, it'll simply inherits from the base class. No boilerplate 😃